apps/www/content/docs/components/stepper.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Stepper
+description: Display content divided into a steps sequence.
+component: true
+---
+
+<ComponentPreview name="stepper-demo" />
+
+## Installation
+
+<Tabs defaultValue="cli">
+
+<TabsList>
+  <TabsTrigger value="cli">CLI</TabsTrigger>
+  <TabsTrigger value="manual">Manual</TabsTrigger>
+</TabsList>
+<TabsContent value="cli">
+
+<Steps>
+
+<Step>Run the following command:</Step>
+
+```bash
+npx shadcn-ui@latest add stepper
+```
+
+<Step>Use the Stepper component with useStepper hook</Step>
+
+```tsx title="page.tsx" {1,9}
+"use client"
+
+import { useStepper } from "@/components/ui/use-stepper"
+
+export default function Page() {
+  const { currentStep, nextStep, prevStep, ...rest } = useStepper()
+  return (
+    // ...
+  )
+}
+```
+
+</Steps>
+
+</TabsContent>
+
+<TabsContent value="manual">
+
+<Steps>
+
+<Step>Copy and paste the following code into your project.</Step>
+
+<ComponentSource name="toast" />
+
+<Step>Update the import paths to match your project setup.</Step>
+
+<Step>Use the Stepper component with useStepper hook</Step>
```

Comment:
@damianricobelli seems `useStepper` source code is missing in this manual installation section